app/globals.css

```
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com');
```

Stylelint (17.3.0)

[error] Expected "url('https://fonts.googleapis.com')" to be "'https://fonts.googleapis.com'" (import-notation)

(import-notation)